src/extensions/step_handlers/swupdate_handler_v2/tests/testdata/adu-yocto-ab-rootfs-update/payloads/example-a-b-update.sh

```diff
@@ -222,9 +222,10 @@ print_help() {
     echo "==========================="
     echo ""
     echo "--workfolder              A work-folder (or sandbox folder)."
-    echo "--swu-file                A swu file to install."
+    echo "--swu-file, --image-file  An image file (.swu) file to install."
```

Comment:
makes it seem like you can use --swu-file OR --image-file but I am not seeing the `--image-file)` below. I only see  `--swu-file)` but it uses `$image_file` instead of `$swu_file`